packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -55,18 +55,18 @@ import { ContainerRuntime } from "./containerRuntime";
 // Snapshot Format Version to be used in store attributes.
 export const currentSnapshotFormatVersion = "0.1";
 
-const attributesBlobKey = ".component";
+export const attributesBlobKey = ".component";
+export const gcBlobKey = ".gc";
```

Comment:
I wonder if this data should be part of .component.
The reason for this - I'm looking at perf of loading, and number of nodes (trees and blobs) in snapshot is one of the contributing factors to size of the snapshot (files like Chapter 1 has over 50% of their size in describing structure of the tree, i.e. it's proportional to number of nodes in tree). The biggest impact is DDSs (as there are more of them than data stores), but data store and .components are big factor (table 20x50 has 1002 components!)

Reply:
That all said, if we assume that such blobs will be big, then it's better to have them as separate blobs, as they may be excluded from initial snapshot. With that, it's better not to name it with dot, as dot is a signal to SPO to include it in initial snapshot payload always. If it's called something else, it will be in snapshot if snapshot is small, and will not (and summarizer will load it separately on demand).

Reply:
These blobs can be really big. I will remove the dot from it.